geode-core/build.gradle

```diff
@@ -360,6 +360,7 @@ dependencies {
   testCompile('org.powermock:powermock-module-junit4')
   testCompile('org.powermock:powermock-api-mockito2')
   testCompile('pl.pragmatists:JUnitParams')
+  testCompile('com.tngtech.archunit:archunit-junit4:0.10.2')
```

Comment:
I think dependency version info is supposed to go in DependencyConstraints.groovy these days.

Reply:
Also, while we're pretty far from there in general, you might want to use `testImplementation` since `compile` directives are deprecated.

Reply:
ok @bschuchardt and @PurelyApplied—it's fixed